internal/proto/controller/api/resources/scopes/v1/scope.proto

```diff
@@ -85,3 +85,28 @@ message Scope {
   // Output only. The authorized actions for the scope's collections.
   map<string, google.protobuf.ListValue> authorized_collection_actions = 310 [json_name = "authorized_collection_actions"];
 }
+
+// Key contains all fields related to a Key in a Scope.
+message Key {
+  // Output only. The ID of the Key.
+  string id = 10; // @gotags: `class:"public"`
+
+  // Output only. The ID of the Scope this Key is in.
+  string scope_id = 20 [json_name = "scope_id"]; // @gotags: `class:"public"`
+
+  // Output only. Scope information for this resource.
+  ScopeInfo scope = 30;
+
+  // Output only. The purpose of the Key.
+  string purpose = 40; // @gotags: `class:"public"`
+
+  // Output only. The time this Key was created.
+  google.protobuf.Timestamp created_time = 50 [json_name = "created_time"]; // @gotags: `class:"public"`
```

Comment:
I think we discussed this when doing all the other classification - because _some_ well known types have special handling ([google.protobuf.StringValue and google.protobuf.BytesValue](https://github.com/hashicorp/go-eventlogger/blob/15f6d722dbe38c79d69680e4f49ad95a1e60cf5e/filters/encrypt/filter.go#L382-L386)), we added classification to all of them.